lib/galaxy/webapps/galaxy/api/histories.py

```diff
@@ -139,6 +140,17 @@ class DeleteHistoryPayload(BaseModel):
     )
 
 
+class DeleteHistoriesPayload(BaseModel):
+    ids: Annotated[List[HistoryIDPathParam], Field(title="IDs", description="List of history IDs to be deleted.")]
```

Comment:
```suggestion
    ids: Annotated[List[DecodedDatabaseIdField], Field(title="IDs", description="List of history IDs to be deleted.")]
```
You probably want to use this instead, the other one is same type but contains annotation information for Path parameters which is unrelated (the same for `UndeleteHistoriesPayload` below).